direct-auth/OktaDirectAuth/OktaDirectAuth/Assets.xcassets/facebook-logo.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "facebook-logo.svg",
```

Comment:
Logo is taken from open [FB source](https://en.facebookbrand.com/facebookapp/assets/f-logo/).